docs/src/main/asciidoc/deploying-to-kubernetes.adoc

```diff
@@ -1268,180 +1268,193 @@ The provided replicas <1>, labels <2> and environment variables <3>  were retain
+
+The required communication for obtaining this information is managed by the Service Binding Operator, which also determines:
 
-Currently, the following Quarkus extensions (and extensions that depend on these) support this feature:
+* how a service consumer is intended to bind to such a service
+* tools for application and service binding, such as Service Binding Operator
 
-* quarkus-jdbc-mariadb
-* quarkus-jdbc-mssql
-* quarkus-jdbc-mysql
-* quarkus-jdbc-postgresql
-* quarkus-kafka-client
-* quarkus-smallrye-reactive-messaging-kafka
-* quarkus-mongo-client
 
-This list of extensions will grow as more services with supported bindings become available on Kubernetes.
+Quarkus supports binding services to applications using the link:https://github.com/k8s-service-bindings/spec[Service Binding Specification for Kubernetes].
+Specifically Quarkus implements the link:https://github.com/k8s-service-bindings/spec#workload-projection[Workload Projection] part of the specification, thus allowing applications running in appropriately configured Kubernetes clusters to consume service, such as a Database or a Broker, without the need for user configuration.
 
 To enable Service Binding support, in addition to one of the currently supported extensions, the `quarkus-kubernetes-service-binding` extension needs to be added to the application dependencies.
 
-=== Example using PostgreSQL
+=== Workload projection
 
-If the Kubernetes cluster in which your Quarkus application runs supports provisioning a PostgreSQL database, one could use the following dependencies to take advantage of the Service Binding feature:
+Workload Projection is the process of projecting binding information to a specific workload, and a way of externalizing configuration. The projection is usually applied as a directory structure that follows certain conventions and can be added to an application or a service using volume mounts. Afterwards, thanks to the `kubernetes-service-binding` extension, the projective binding can be used as a configuration source, allowing the configuration of additional modules such as:
 
-[source,xml,role="primary asciidoc-tabs-target-sync-cli asciidoc-tabs-target-sync-maven"]
-.pom.xml
-----
-<dependency>
-    <groupId>io.quarkus</groupId>
-    <artifactId>quarkus-jdbc-postgresql</artifactId>
-</dependency>
-<dependency>
-    <groupId>io.quarkus</groupId>
-    <artifactId>quarkus-kubernetes-service-binding</artifactId>
-</dependency>
+* databases
+* message brokers
 
-<!-- Hibernate isn't necessary for Service Binding, but most relational database access will benefit from using it -->
-<dependency>
-    <groupId>io.quarkus</groupId>
-    <artifactId>quarkus-hibernate-orm</artifactId>
-</dependency>
-----
+Users often use this concept of workload projection during the development time to connect their application to a dev-database, or other locally runned services, without changing the actual application code or configuration. 
```

Comment:
Where does this come from? I personally don't know anyone that does this...

Reply:
I have got this from my latest meeting with Ioannis. @iocanel 
Maybe I just misunderstood something but thats what we got in the google doc

Reply:
Also, I have an upgrade for this:

During application development, users can use workload projection to connect their application to a dev-database, or other locally-run services, without changing the actual application code or configuration.

Reply:
This comes from me. I am not sure how extensively this pattern is used, but my understanding is that one of the benefits of workload projection compared to the traditional evn variable style of binding is the fact that it can be simpler to use in development, testing etc.

Reply:
.. and the other being security

Reply:
Yes, I added it since I find it valuable for the section.